src/tests/multihost/ad/test_adparameters.py

```diff
@@ -756,3 +756,52 @@ def test_0018_bz1734040(multihost, adjoin, create_aduser_group):
         client.restore_sssd_conf()
 
         assert find.search(log), "Expected log record is missing."
+
+    @pytest.mark.tier1
+    def test_0019_bz2070189(self, multihost, adjoin):
+        """
+        :title: sssd error triggers backtrace :
+        [write_krb5info_file_from_fo_server] (0x0020): [RID#73501]
+        There is no server that can be written into kdc info file. is set
+        and files provider is also enabled
+        :id:
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2070189
+        :steps:
+          1. setup AD environment
+          2. Join linux machine into AD domain
+          3. Set the debug_level in sssd.conf for AD domain to 1,
+            make sure backtrace dump is enabled (debug_backtrace_enabled=True)
+            make sure that krb5_use_kdcinfo=True
+          4. clear the cache sss_cache -E
+          5. lookup AD user `id administrator@domain`
+        :results:
```

Comment:
This should be `:expectedresults:` and the number of results should match the number of steps for it to get added in polarion in a proper tabular format

Reply:
Does the importer allow multiple lines per step like for step 3 above?

Reply:
Yes, it allows, but we either have to add a `\` at the end of the line or just make sure the next line is indented to start from right under the top line. Second option is cleaner.
eg.
```
            2. Join linux machine into AD domain
            3. Set the debug_level in sssd.conf for AD domain to 1,
               make sure backtrace dump is enabled,
               debug_backtrace_enabled=True
               make sure that krb5_use_kdcinfo=True
            4. clear the cache sss_cache -E
```